Maddori.Apple/Maddori.Apple/Global/UIComponent/View/AlertView.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  AlertView.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/11/06.
+//
+
+import UIKit
+
+import SnapKit
+
+enum AlertType: String {
+    case delete = "삭제하기"
+    case join = "합류하기"
+    
+    var title: String {
+        switch self {
+        case .delete:
+            return TextLiteral.feedbackFromMeDetailViewControllerAlertViewTitle
+        case .join:
+            return "합류 할 팀"
+        }
+    }
+    
+    var subTitle: String {
+        switch self {
+        case .delete:
+            return TextLiteral.feedbackFromMeDetailViewControllerAlertViewSubTitle
+        case .join:
+            return "팀에 합류하시겠어요?"
+        }
+    }
+    
+    var actionTitle: String {
+        switch self {
+        case .delete:
+            return "삭제"
+        case .join:
+            return "참여"
+        }
+    }
+    
+    var actionTitleColor: UIColor {
+        switch self {
+        case .delete:
+            return .red100
+        case .join:
+            return .gray500
+        }
+    }
+}
+
+final class AlertViewController: BaseViewController {
+    
+    var type: AlertType
+    var teamName: String? = nil
+    
+    init(type: AlertType) {
+        super.init()
+        self.type = type
+    }
+    
+    init(type: AlertType, teamName: String?) {
+        super.init()
+        self.type = type
+        self.teamName = teamName
+    }
```

Comment:
p1;

```suggestion
    init(type: AlertType) {
        self.type = type
        super.init()
    }
    
    init(type: AlertType, teamName: String?) {
        self.type = type
        self.teamName = teamName
        super.init()
    }
```

현재 빌드 에러가 나는 상태네요
해달 클래스의 변수들이 설정 되고 난 이후에 init되어야 합니다 !

Reply:
[c2ec8e4](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/78/commits/c2ec8e48a5f8b45d974a1b2ae5937443f0878086)
왁 수정해서 반영해두었습니다! 
[✔️] 빌드하고 푸시하는 습관 기르기..! 감사합니당!